seqr/utils/elasticsearch/es_utils_tests.py

```diff
@@ -451,6 +453,189 @@
     'matched_queries': {SV_WGS_INDEX_NAME: ['F000014_14']},
   }
 
+ES_MITO_WGS_VARIANT = {
+    "_source" : {
+      "genotypes" : [
+        {
+          "num_alt": 2,
+          "gq": 60.0,
+          "hl": 1.0,
+          "mito_cn": 319.03225806451616,
+          "contamination": 0.0,
+          "dp": 5139.0,
+          "sample_id": "MCAI043"
+        },
+      ],
+      "samples_gq_0_to_5" : [ ],
+      "samples_gq_5_to_10" : [ ],
+      "samples_gq_10_to_15" : [ ],
+      "samples_gq_15_to_20" : [ ],
+      "samples_gq_20_to_25" : [ ],
+      "samples_gq_25_to_30" : [ ],
+      "samples_gq_30_to_35" : [ ],
+      "samples_gq_35_to_40" : [ ],
+      "samples_gq_40_to_45" : [ ],
+      "samples_gq_45_to_50" : [ ],
+      "samples_gq_50_to_55" : [ ],
+      "samples_gq_55_to_60" : [ ],
+      "samples_gq_60_to_65" : [
+        "MCAI043"
+      ],
+      "samples_gq_65_to_70" : [ ],
```

Comment:
empty fields like this should not be exporting to elasticsearch. If they are thats a bug you should fix, and either way you shouldn't include them in the fixture data

Reply:
Agree. The empty fields do be exported to ES. I can create a PR to fix the pipeline. But the SNP pipeline will be changed as well.

Reply:
The SNP pipeline already doesn't export these fields. Are you sure the mito pipeline does?